var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -125,7 +125,7 @@ class Mesa(MesonPackage):
     # requires native to be added to llvm_modules when using gallium swrast
     patch('https://cgit.freedesktop.org/mesa/mesa/patch/meson.build?id=054dd668a69acc70d47c73abe4646e96a1f23577', sha256='36096a178070e40217945e12d542dfe80016cb897284a01114d616656c577d73', when='@21.0.0:21.0.3')
 
-    patch('mesa_check_llvm_version_suffix.patch', when='@21.2.3:')
+    patch('handle_missing_set_override_stack_alignment.patch', when='@21.2.3:')
```

Comment:
Can you add a comment above the patch to briefly explain what the patch does, why is it needed and (if possible) where it was taken from?

Reply:
I wrote the patch. I've added a description in 931309914d31a2d6b7e80615a53a47a3ed86f82a. This sample illustrates what it does:  https://godbolt.org/z/jEdq4GTW1. In the top code sample, `llvm::Module::setOverrideStackAlignment` exists and thus `try_set_override_stack_alignment` is defined as calling it. In the bottom sample, `llvm::Module::setOverrideStackAlignment` does not exist and `try_set_override_stack_alignment` is a noop.

Reply:
Thanks, the comment LGTM